src/helpers/tests/artifacts/user.gen.ts

```diff
@@ -6,6 +6,8 @@
 // NOTE: ANY CHANGES MADE WILL BE OVERWRITTEN ON SUBSEQUENT EXECUTIONS OF MONGOOSE-TSGEN.
 
 import mongoose from "mongoose";
+ import { Prop, Schema } from '@nestjs/mongoose';
```

Comment:
I want to comment on things like this indent, and lack of a newline between @Schema and @Prop items... but will that be resolved with a simple save, with a run of Prettier?

Reply:
Correct. I'd like to fix the spacing issues but it's very low priority for that reason.